test/e2e/api_connectors_test.go

```diff
@@ -97,10 +96,12 @@ var _ = Context("Payments API Connectors", func() {
 			err := ConnectorInstall(ctx, app.GetValue(), ver, connectorConf, &connectorRes)
 			Expect(err).To(BeNil())
 
-			delRes := struct{ Data string }{}
+			delRes := struct {
+				Data models.Task `json:"data"`
+			}{}
 			err = ConnectorUninstall(ctx, app.GetValue(), ver, connectorRes.Data, &delRes)
 			Expect(err).To(BeNil())
-			Expect(delRes.Data).To(Equal(connectorRes.Data))
+			Expect(delRes.Data.ID.String()).ToNot(BeEmpty())
```

Comment:
Since we have a task returned, you can now use the `TaskPoller`to fetch the Task until it gets processed. You can see examples of that in `test/e2e/api_payment_initiations_test.go`

Reply:
I just added them thank you for the tips !